doc/flame/examples/lib/post_process.dart

```diff
@@ -0,0 +1,69 @@
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:doc_flame_examples/ember.dart';
+import 'package:flame/components.dart';
+import 'package:flame/game.dart';
+import 'package:flame/post_process.dart';
+import 'package:flutter_shaders/flutter_shaders.dart';
+
+class PostProcessGame extends FlameGame {
+  @override
+  Future<void> onLoad() async {
+    await super.onLoad();
+
+    world.add(
+      PostProcessComponent(
+        postProcess: PixelationPostProcess(),
+        position: Vector2(0, 0),
+        anchor: Anchor.center,
+        children: [
+          EmberPlayer(position: Vector2(0, 0), size: Vector2(100, 100)),
```

Comment:
```suggestion
          EmberPlayer(size: Vector2(100, 100)),
```
Position is 0,0 by default.